Library/Formula/libspnav.rb

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
@DomT4 I think GitHub messed up with the diff and your comment isn’t relevant anymore. Is it?